runner/materialize.go

```diff
@@ -15,8 +15,9 @@ import (
 	"github.com/featureform/types"
 )
 
-const MAXIMUM_CHUNK_ROWS int64 = 1024
-const WORKER_IMAGE string = "featureformcom/worker"
+const MAXIMUM_CHUNK_ROWS int64 = 102400
```

Comment:
Will this work with 1 million rows? If we have a 10M row dataset this is going to spawn 100 jobs